tests/product/base_product_case.py

```diff
@@ -324,12 +324,19 @@ def assert_file_content(self, host, filepath, expected):
             expected)
 
     def file_content_message(self, actual, expected, pa_file):
-        msg = '%s != %s' % (actual, expected)
+        msg = '\t===== vv ACTUAL FILE CONTENT vv =====\n' \
+              '%s\n' \
+              '\t=========== DID NOT EQUAL ===========\n' \
+              '%s\n' \
+              '\t==== ^^ EXPECTED FILE CONTENT ^^ ====\n' \
+              '' % (actual, expected)
         if pa_file:
             try:
-                msg += '\n Content for presto-admin file %s \n' % pa_file
+                msg += '\t==== Content for presto-admin file %s ====\n' % \
+                       (pa_file,)
                 msg += self.get_file_content(self.cluster.get_master(),
```

Comment:
Sorry, this is confusing and I should have added a comment earlier. There's one now. TL;DR: the master is the correct host, we're getting the file as it exists on the presto-admin host as a sanity check of the actual file content.